### PR TITLE
Refactor: writeConfiguration

### PR DESCRIPTION
**Pull request by haslinghuis**
Functions have been refactored to be defined in one place only:

`MspHelper.prototype.writeConfiguration = function(bool reboot)`

Add a new mspHelper for `MSP_EEPROM_WRITE` and optional `MSP_SET_REBOOT` to finish callback `reinitializeConnection`.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
Approved, but for next PRs I would like to have some kind of description in the PR. Latelly we are letting it empty and some PRs, like this, are not basic and a description/explanation helps.

**Comment by haslinghuis** [comment]
@McGiverGim added description.

**Comment by nerdCopter** [comment]
`34b6c04a`

the following tabs have unexpected reboot, upon `save`.
* Power
* Modes
* Video Transmitter
* LED

**Review by nerdCopter — APPROVED** [review]
* `13c07de8` works overall for unsoldered FC.
* might suggest others test further.

**Pull request merged**
